src/controller.cpp

```diff
@@ -388,8 +397,7 @@ void Controller::activateInternal(DNSPortPolicy dnsPort,
   exitConfig.m_serverPublicKey = exitServer.publicKey();
   exitConfig.m_serverIpv4AddrIn = exitServer.ipv4AddrIn();
   exitConfig.m_serverIpv6AddrIn = exitServer.ipv6AddrIn();
-  exitConfig.m_serverPort = exitServer.choosePort();
```

Comment:
Just verifying we don't need this? I see we're setting it again further down but just checking.

Reply:
Ahhh thank you! that is still needed